### PR TITLE
New package: Microsoft.Office.SubjectInterfacePackages version 16.0.16507.43425

### LINKED ISSUE #152520: [Package Request]: Microsoft Office Subject Interface Packages for Digitally Signing VBA Projects
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.microsoft.com/en-us/download/details.aspx?id=56617

Publisher: Microsoft

Package Name: 
Microsoft Office Subject Interface Packages for Digitally Signing VBA Projects

Description: 
- This download includes the Subject Interface Packages (SIPs) that enable the digital signing and signature verification of Visual Basic for Applications projects within most Microsoft Office file formats that support VBA macros.
- This download includes two Subject Interface Package (SIP) libraries that support the digital signing and signature verification of Visual Basic for Applications projects within most Office file formats that support VBA macros. One of the SIPs supports the Open Office XML file formats, and the other supports all other supported file formats. A SIP is a library that integrates with the Windows Operating System cryptographic stack to enable the OS to perform cryptographic operations on file formats for which the OS is not aware by default.
- Once installed, existing tools and code that utilize the Windows cryptographic APIs to sign and verify files can be additionally used to sign and verify Office files containing VBA projects.

--> Important. Currently this package does not auto register its libraries.

Package Version:16.0.16507.43425

Installer URL:https://download.microsoft.com/download/F/B/4/FB46F8CA-6A6F-4CB0-B8F4-06BF3D44DA48/officesips_16.0.16507.43425.exe

### PR DESCRIPTION
**Pull request by schembriaiden**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.1 :rocket:

Resolves: https://github.com/microsoft/winget-pkgs/issues/152520
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/152521)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @vikingnope, 

This package has an unusual install flow:
- Portable exe is actually installer file - running it prompts for license agreement: 
image
  - Then an installer location:
image
  - The installer switch `/?` gave: 
image
- After that installer completes, the installed EXE has a dependency:
image

The outer install is okay currently. The `/quiet` switch performs the nested install without user interaction (`/passive` leaves a messagebox to click "OK"). And the nested installer needs a `Custom` switch with `/extract:C:\path\to\extraction\location`, or the installer will silently fail (or maybe install to an unknown location?)

**Comment by stephengillie** [comment]
Should this have a different or more clear `PackageIdentifier`? My brain expands the acronym "SIP" into "Session Initiation Protocol" so I keep thinking this is a VOIP function for MS Office.

**Comment by Trenly** [comment]
> This package has an unusual install flow:

It's also worth noting that this package doesn't write any ARP Entries, so WinGet won't be able to determine if it is installed.

> And the nested installer needs a `Custom` switch with `/extract:C:\path\to\extraction\location`

The `InstallLocation` switch could be added and `InstallLocationRequired: true` can be set in the manifest

File: manifests/m/Microsoft/Office/SubjectInterfacePackages/16.0.16507.43425/Microsoft.Office.SubjectInterfacePackages.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+PackageIdentifier: Microsoft.Office.SubjectInterfacePackages
+PackageVersion: 16.0.16507.43425
+InstallerType: exe
+InstallModes:
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet            
+  SilentWithProgress: /quiet
+  InstallLocation: /extract:C:\path\to\extraction\location
```
Comment (Trenly) [inline manifests/m/Microsoft/Office/SubjectInterfacePackages/16.0.16507.43425/Microsoft.Office.SubjectInterfacePackages.installer.yaml:13]:
```suggestion
  InstallLocation: /extract:<INSTALLPATH>
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\WinGetInstallerRoot\Microsoft.Office.SubjectInterfacePackages\offclearsig.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean Dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 880.)

**Comment by vikingnope** [comment]
I think it was the wrong package, I think it should be correct now.

**Comment by vikingnope** [comment]
@stephengillie, can you recheck this please?

**Comment by stephengillie** [comment]
This is unchanged.

> This updated manifest doesn't seem to install the dependency.
image

**Pull request closed without merging**